### DISCUSSION TITLE
Oops, that's a four-oh-four.

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#238] [Q&A] [answered]
getting this error how can i slove this? [link](https://onedrive-vercel-index-silk-sigma.vercel.app/)

**Comment by spencerwooo** [comment, marked as answer]
i believe your error is the same with https://github.com/spencerwooo/onedrive-vercel-index/discussions/240

i cannot guarantee solutions, but check whether you have:

- configured your `site.json`, where the `baseDirectory` should be a folder in your root OneDrive path
- the redis db url (if hosted on upstash, and has SSL enabled), should be something like `rediss://:xxx@us1-xxx.upstash.io:xxx`
  - _notice the `rediss` instead of `redis` and the `://:xx` instead of `://xx`_

if you can do an `npm run dev` locally and check for debug logs that would also be helpful for solving your issue.

**Reply by hirockstar** [reply]
it there any video of depoly this project it will be more great for us.
